src/Models/Users.php

```diff
@@ -873,4 +873,25 @@ public function unsubscribe(int $notificationTypeId) : NotificationsUnsubscribe
         $systemModulesId = $notificationType ? $notificationType->system_modules_id : -1;
         return Notifications::unsubscribe($this, (int) $notificationTypeId, (int) $systemModulesId);
     }
+
+    /**
+     * Desassociated a user from an app
+     * 
+     * @return bool
+     */
+    public function disassociateFromApp() : bool
```

Comment:
@rwhite27 move this to the model userAssociateApps
- Pass it the user and company so we can use this on other locations